client/components/chart/style.scss

```diff
@@ -0,0 +1,222 @@
+/** @format */
+.woocommerce-dashboard__widget-chart {
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+	align-items: flex-start;
+	margin: -20px;
+	border-top: 0.5px solid rgba(226, 228, 231, 0.7);
+
+	.woocommerce-dashboard__widget-chart-header {
+		min-height: 50px;
+		border-top: 0.5px solid rgba(226, 228, 231, 0.7);
+	}
+
+	.woocommerce-dashboard__widget-chart-body {
+		display: flex;
+		flex-direction: row;
+		justify-content: flex-start;
+		align-items: flex-start;
+		width: 100%;
+
+		&.woocommerce-dashboard__widget-chart-body-column {
+			flex-direction: column;
+		}
+	}
+
+	.woocommerce-dashboard__widget-chart-footer {
+		width: 100%;
+
+		.woocommerce-legend {
+			&.woocommerce-legend__direction-column {
+				height: 100%;
+				min-height: none;
+			}
+		}
+	}
+
+	.key-colour {
+		width: 10px;
+		height: 10px;
+		margin-right: 8px;
```

Comment:
Where possible, we should be using the spacing variables in `stylesheets/_variables.scss`. Some margins/padding you have in this file (6px) should be updated to $gap-smallest * 2.